src/sass/components/_footer.scss

```diff
@@ -0,0 +1,77 @@
+.footer {
+  background-color: $pageBg;
+  display: block;
+  flex-direction: column;
+  min-width: 320px;
+
+  &__header {
+    color: $brand-pink;
+    font-family: 'Titan One';
+    font-size: 18px;
+    font-weight: 400;
+    display: block;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding-top: 20px;
+    padding-left: 20px;
+    max-width: 80%;
+    &--color {
+      color: $brand-red;
+      font-size: 34px;
+      margin-top: 16px;
+      display: block;
+    }
+  }
+  &__list {
+    display: flex;
+    list-style: none;
+    align-items: flex-end;
+    justify-content: flex-end;
+    padding-right: 20px;
+  }
+  &__circle {
+    padding: 3px;
+    background-color: $brand-pink;
+    border-radius: 50%;
+    fill: $white;
+    display: flex;
+    width: 100%;
+
+    &:hover {
+      background-color: $brand-red;
+    }
+  }
+  &__link {
+    justify-content: flex-end;
+    align-items: flex-end;
+    margin-left: 28px;
+    margin-top: 43px;
```

Comment:
Nie powinnaś odsuwać elementu od elementu elementem rodzica. Czyli jeśli chcesz odsunąć h2 od ul to dajesz margin na h2 albo na ul, nigdy na dzieciach tych tagów tak jak zrobiłaś teraz umieszczając na dziecku ul czyli li

Reply:
Ale margin-left był po to, żeby odsunąć elementy listy od siebie, także też wywalić? zresztą, po wywaleniu flexa i tak się to rozjeżdża, bo przecież domyślnie ma block, nie flex.